pyroSAR/gamma/util.py

```diff
@@ -655,29 +703,35 @@ def geocode(scene, dem, tmpdir, outdir, targetres, scaling='linear', func_geobac
             print('orbit state vector correction failed for scene {}'.format(scene.scene))
             return
     
+    print('calibrating...')
     calibrate(scene, scene.scene, logpath=path_log, outdir=scene.scene, shellscript=shellscript)
-    
-    images = [x for x in scene.getGammaImages(scene.scene) if x.endswith('_grd') or x.endswith('_slc_cal')]
-    
+    images = [x for x in scene.getGammaImages(scene.scene) if x.endswith('_grd') or x.endswith('_slc_cal') or x.endswith('_mli_cal')] 
+         
     products = list(images)
-    
-    print('multilooking..')
-    for image in images:
-        multilook(infile=image, outfile=image + '_mli', targetres=targetres,
-                  logpath=path_log, outdir=scene.scene, shellscript=shellscript)
-    
-    images = [x + '_mli' for x in images]
-    products.extend(images)
+
+    if scene.sensor in ['S1A', 'S1B']:
```

Comment:
why no multilooking for PALSAR?

Reply:
the path data is already multi-looked and the _mli is included in the file name (*_mli_cal). Re-multi-looking results in *_mli_cal_mli file name ending and the images would be doubled in the images list in case of "original" resolution applied as targetres.